packages/react/src/search/search.react.tsx

```diff
@@ -0,0 +1,22 @@
+import { c, classy } from '@onfido/castor';
+import { Icon } from '@onfido/castor-react';
+import React from 'react';
+import { Input, InputProps } from '../input/input.react';
+import { withRef } from '../utils';
+
+/**
+ * @note `Search` uses an `Icon` that requires
+ * `Icons` (SVG sprite) to be included in your app.
```

Comment:
To limit to 80 characters:

```suggestion
 * @note `Search` uses an `Icon` that requires `Icons` (SVG sprite) to be
 * included in your app.
```

Also, this comment style is different from the one on `Icon` component - mind amending that one too? Just to keep consistency.

Reply:
Ah I didn't notice Icon had a similar one, fixed.